packages/angular/migrations.json

```diff
@@ -262,6 +267,23 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "11.1.0": {
+      "version": "11.1.0-beta.0",
```

Comment:
I guessed at this being a good upcoming point for this

Reply:
Yup. This is perfect